index.html

```diff
@@ -43,7 +43,7 @@ <h2>Cat Form</h2>
           <form action="https://freecatphotoapp.com/submit-cat-photo">
             <fieldset>
              
-              <label><input id="indoor" type="radio" name="indoor-outdoor" value="indoor" checked> Indoor</label>
+              <label><input id="indoor" type="radio" name="indoor-outdoor"  checked> Indoor</label>
```

Comment:
In the label element, the nested <input> element should have a value attribute and  assigned a value for it,

Reply:
- [ ] Great work making this comment. However, the comment sounds like a command and you did not give reasons why the author should add a `value` attribute to the nested `input` element. It is important to be polite in your review. 

Being polite in reviews will lead to a conversation, instead of receiving an opposing response or no response at all.

Giving clear and specific reasons help the author of the code to understand the intent behind the suggested changes, fostering better communication and reducing ambiguity. This allows for a more constructive dialogue between team members. 👍 

#### Example of Polite Suggestion:

>  "I think it would be better to use semantic tags"
> "Could we try ...."

#### Eample of Comment with Reasons 

> Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

#### Hint: I would advise you to edit your comment above and implement the suggestions I just gave. 😎